Homework 5

VadymDemchenko opened the pull request:
Here is my homework for lesson 5

VadymDemchenko committed:
cleaning branch homework_5

VadymDemchenko committed:
homework_5 added

klymenok commented on HW05/homework_5.py:
```diff
@@ -0,0 +1,28 @@
+        print("Це непарне число")
+elif user_input.replace(' ', '').isalpha():
+
+    list_user_input = list(user_input)
```
There is no point of using this transformation. You can use a string in for-in

klymenok commented on HW05/homework_5.py:
```diff
@@ -0,0 +1,28 @@
+upper_text = 0
+lower_text = 0
+spaces = 0
+if user_input.isdigit():
```
The task was to loop over every symbol from input string. This means that you should have one for-in loop and inside of it you should check every symbol

klymenok reviewed (changes requested):
Some changes are required

VadymDemchenko committed:
Update homework_5.py

2 issues have been fixed. Waiting for your feedback

klymenok reviewed (changes requested):
There is still one unresolved comment. Please, check it

VadymDemchenko committed:
Update homework_5.py

Sorry, it seems I didn't understand task right. Now it works in right way (I hope on it)
Thanks for your feedback

klymenok reviewed (approved):
Looks good!
